subsys/net/l2/wifi/Kconfig

```diff
@@ -27,3 +27,10 @@ config WIFI_MGMT_RAW_SCAN_RESULTS_ONLY
 	  to the application.
 
 endif # WIFI_MGMT_RAW_SCAN_RESULTS
+
+config WIFI_MGMT_FORCED_PASSIVE_SCAN
```

Comment:
Why would we need an option to enforce certain type of scan, somewhat hidden internally from the user?

Reply:
In general, Wi-Fi chips do below:
* Passive scan - only when enforced by regulatory 
* Active scan - all other cases (to save time)

But in lieu of power consumption this global override helps, this was a request from couple of customers.

Reply:
Ok, thanks for the clarification.